src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -59,6 +59,9 @@ public class CommandTestUtil {
     public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
     public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
 
+    public static final String INVALID_EMAIL = "test@example.com";
+    public static final String INVALID_STUDENT_ID = "A2222222A";
```

Comment:
Just checking, why are these considered invalid email and student id? Perhaps you could add a comment to point out the reasoning?

Reply:
Definitely could have done better on the naming convention. These are valid in format but invalid for the purpose of the test case (i.e there does not exist a person with such email/id)